test/file-uploader.spec.js

```diff
@@ -151,7 +151,7 @@ describe('fileUploader', () => {
 
     const axiosPostStub = sandbox.stub(axios, 'post').callsFake((url, data) => {
       return new Promise((resolve, reject) => {
-        const measurementSetBody = JSON.parse(data);
+        const measurementSetBody = JSON.parse(JSON.stringify(data));
```

Comment:
do we need to stringify and parse this? Wouldn't that mean it is already an object?